docs/guides/server/logging.adoc

```diff
@@ -188,10 +189,79 @@ To configure a different logging format for the file log handler, enter the foll
 
 See <<Configuring the console log format>> for more information and a table of the available pattern configuration.
 
+== Centralized logging using Syslog
+
+{project_name} provides the ability to send logs to a remote Syslog server.
+It utilizes the protocol defined in https://datatracker.ietf.org/doc/html/rfc5424[RFC 5424].
+
+=== Enable the Syslog handler
+To enable logging using Syslog, add it to the list of activated log handlers as follows:
+
+<@kc.start parameters="--log=\"console,syslog\""/>
+
+=== Configuring the Syslog endpoint
+
+To configure the endpoint(_host:port_) of your centralized logging system, enter the following command and substitute the values with your specific values:
```

Comment:
One small typo.
```suggestion
To configure the endpoint (_host:port_) of your centralized logging system, enter the following command and substitute the values with your specific values:
```

Reply:
Oops, auto-merge was on. Can be done in a different PR.